packages/playground/metro.config.js

```diff
@@ -67,7 +72,6 @@ module.exports = {
           .replace(/[/\\\\]/g, '[/\\\\]')}.*`,
       ),
     ]),
-    hasteImplModulePath: path.resolve(__dirname, 'hasteImpl.js'),
```

Comment:
>hasteImplModulePath: path.resolve(__dirname, 'hasteImpl.js'), [](start = 4, length = 61)

Did you mean to remove this?  Looks like something for 61.

Reply:
It is accident, but I kind of feel this is not needed anymore event for 60. But I will put it back and create a issue on our 61 project board to visit all hasteImpl.js.

---
In reply to: [370431313](https://github.com/microsoft/react-native-windows/pull/3944#discussion_r370431313) [](ancestors = 370431313)